src/EA.Iws.Web.Tests.Unit/Controllers/AdminExportNotificationMovements/CaptureMovementControllerTests.cs

```diff
@@ -0,0 +1,183 @@
+﻿namespace EA.Iws.Web.Tests.Unit.Controllers.AdminExportNotificationMovements
+{
+    using System;
+    using System.Threading.Tasks;
+    using Areas.AdminExportNotificationMovements.Controllers;
+    using Areas.AdminExportNotificationMovements.ViewModels.CaptureMovement;
+    using Core.Movement;
+    using Core.Shared;
+    using FakeItEasy;
+    using Prsd.Core.Mediator;
+    using Requests.NotificationMovements.Capture;
+    using Web.Infrastructure;
+    using Web.Infrastructure.Authorization;
+    using Xunit;
+
+    public class CaptureMovementControllerTests
+    {
+        private readonly IMediator mediator;
+        private readonly CaptureMovementController controller;
+        private readonly Guid notificationId = new Guid("AF81049E-4F94-479A-921F-27B27C4F8BB9");
+        private readonly IAuditService auditService;
+        private readonly AuthorizationService authorizationService;
+        private readonly DateTime receivedDate = new DateTime(2019, 3, 2);
+        private readonly DateTime recoveredDate = new DateTime(2019, 3, 2);
+        private readonly DateTime prenotifiedDate = new DateTime(2019, 3, 2);
+        private readonly DateTime actualDate = new DateTime(2019, 3, 2);
+        private readonly DateTime rejectedDate = new DateTime(2019, 3, 2);
+        private Guid? movementId;
```

Comment:
Build is failing because this field is not being set anywhere so will always have a default value. Same applies for the tests in import notifications.